### ISSUE TITLE
[BUG] TrueNas Scale - All NVMe drives show as failed

### ISSUE THREAD

**Issue by panzerscope** [AnalogJ/scrutiny#721]
**Describe the bug**
The app shows all NVMe drives as FAILED. This is highly unlikely to be the case as at least two of the drives are only 145 days old from the discovery of this bug.

**Expected behavior**
For the NVMe drives to have PASSED, or at minimum for some of them to have passed.

**Information** 

I am running TrueNas Scale ElectricEel-24.10.0.2 and Scrutiny version 0.8.1. During the Scrutiny app installation I initially did NOT have "Warning: Elevate Privileges" enabled which meant even though the NVMe drives were seen, value and readings were missing as seen below. 

![image](https://github.com/user-attachments/assets/8094ae37-79ec-4efb-91d7-79235faf9133)

At this point I edited the settings and enabled "Warning: Elevate Privileges" after which additional values were now being detected, however the drives still showed as failed. as per the below screenshot. 

![image](https://github.com/user-attachments/assets/15bfd465-034a-4d9e-ba82-0e0f25f2c12a)

If logs files are required to assist with diagnostics, please can you instruct me on how to get the appropriate logs using TrueNas.

Many thanks!

**Comment by dspalu32** [comment]
I saw precisely the same set of circumstances on TrueNAS: needing to re-run with elevated privileges to get NVMe drives to read correctly, but then seeing the drive marked as failed despite now having 'good' data.

I simply nuked the the config and db and restarted. All is well. I'm guessing the failure status may have self-cleared after some refresh cycle, but I had no history to loose.

(rm -r /mnt/.ix-apps/app_mounts/scrutiny/config/* /mnt/.ix-apps/app_mounts/scrutiny/influxdb/*)

**Comment by panzerscope** [comment]
> I saw precisely the same set of circumstances on TrueNAS: needing to re-run with elevated privileges to get NVMe drives to read correctly, but then seeing the drive marked as failed despite now having 'good' data.
> 
> I simply nuked the the config and db and restarted. All is well. I'm guessing the failure status may have self-cleared after some refresh cycle, but I had no history to loose.
> 
> (rm -r /mnt/.ix-apps/app_mounts/scrutiny/config/* /mnt/.ix-apps/app_mounts/scrutiny/influxdb/*)

Thanks, Just deleting the config file seems to have fixed the issue. I will create another bug report however as for some reason my images are missing from the app in the noted locations lol 

![image](https://github.com/user-attachments/assets/60a9714a-f973-4795-96fa-b51e51039e8a)

**Comment by kabatp** [comment]
I can confirm that deleting the content of influxdb and config folder in app_mounts on TrueNas Scale 24.10.0.2 with Scrunity v0.8.1 and restarting the container fixed the NVMe issue - all pictures were loaded correctly

**Comment by AnalogJ** [comment]
yeah, scrutiny doesn't yet have a way to reset the device level status from the UI -- though there is a small script in the troubleshooting guide that can help- https://github.com/AnalogJ/scrutiny/blob/master/docs/TROUBLESHOOTING_DEVICE_COLLECTOR.md#device-failed-but-smart--scrutiny-passed
